releases/release-1.31/release-notes/maps/pr-123587-map.yaml

```diff
@@ -1,3 +1,4 @@
 pr: 123587
 releasenote:
   text: Fixed "-kube-test-repo-list" e2e flag may not take effect.
```

Comment:
is the first "-" in "-kube-test-repo-list" a typo?

Reply:
It's what the author originally had: https://github.com/kubernetes/kubernetes/pull/123587